components/forms/w-input.vue

```diff
@@ -1,19 +1,28 @@
 <template>
-  <w-field v-bind="{ ...$attrs, ...$props }" #default="{ triggerValidation, aria }" :class="{ 'has-suffix': $slots.suffix, 'has-prefix': $slots.prefix }">
-    <div class="input" :class="inputWrapperClass">
+  <w-field v-bind="{ ...$attrs, ...$props }" #default="{ triggerValidation, aria }">
+      <div :class="[inputWrapperClass, wrapperClass]">
```

Comment:
Not sure if `inputWrapperClass` prop should be before or after `wrapperClass`, or if it makes no difference 🤔

Reply:
Don't know either, let's reverse so we do not need to wonder 👍

Reply:
Fixed https://github.com/warp-ds/vue/pull/13/commits/7ed22d25bb5843562d61b4e666c20894bfa753f4